pkg/cluster/reconcile.go

```diff
@@ -272,8 +272,8 @@ func RequestBackupNow(httpClient *http.Client, addr string) error {
 	return nil
 }
 
-func checkBackupExist(httpClient *http.Client, addr string) (bool, error) {
-	resp, err := httpClient.Head(fmt.Sprintf("http://%s/backup", addr))
+func checkBackupExist(httpClient *http.Client, addr, ver string) (bool, error) {
+	resp, err := httpClient.Head(fmt.Sprintf("http://%s/backup?version=%s", addr, ver))
```

Comment:
will change to "etcdVersion".

> etcdVersion can be different from backup version.

I will add a TODO here. Let's have a policy docs for this first.